go/healthserver/impl.go

```diff
@@ -0,0 +1,37 @@
+package healthserver
+
+import "net/http"
+
+var (
+	healthStatus string
+	readyStatus  string = "not ready"
+)
+
+func init() {
+	http.HandleFunc("/healthz", healthzHandler)
+	http.HandleFunc("/readiness", readinessHandler)
+}
+
+func commonHandler(w http.ResponseWriter, status string) {
+	if status == "" {
+		w.Write([]byte("OK"))
```

Comment:
Nit pick: You could make []byte("OK") a global variable to ensure you don't get gratuitous memory allocations.